subsys/net/lib/coap/coap.c

```diff
@@ -763,6 +763,17 @@ static bool uri_path_eq(const struct coap_packet *cpkt,
 			continue;
 		}
 
+	if (IS_ENABLED(CONFIG_COAP_URI_WILDCARD) && strlen(path[j]) == 1) {
```

Comment:
This whole block needs to be indented with one extra tab.

Reply:
yes, thanks